care/users/api/serializers/user.py

```diff
@@ -382,6 +377,14 @@ class Meta:
             "skills",
         )
 
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+        instance = User.objects.prefetch_related("skills").get(pk=instance.pk)
+        skills = instance.skills.filter(userskill__deleted=False)
+        skills_data = SkillSerializer(skills, many=True).data
+        representation["skills"] = skills_data
+        return representation
+
```

Comment:
This will still make N+1 queries for a list of users, the most efficient way would be to prefetch these objects where the queryset is being called.

Reply:
Ohh yes , I got it know. I will make the changes